### PR TITLE
cephfs-top: display filters in the header

### PR DESCRIPTION
**Pull request by joscollin**
This would help the user to identify what are the filters currently applied to the cephfs-top screens at a glance.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [x] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: src/tools/cephfs/top/cephfs-top [hunk 1]
```diff
@@ -811,7 +813,9 @@ class FSTop(object):
                                                                  num_mounts=num_mounts,
                                                                  num_kclients=num_kclients,
                                                                  num_libs=num_libs), curses.A_DIM)
-        self.header.addstr(4, 0, help, curses.A_DIM)
+        self.header.addstr(4, 0, f"Filters: Sort={current_states['last_field']}, "
```
Comment (neesingh-rh) [inline src/tools/cephfs/top/cephfs-top:816]: It seems helpful from user's perspective, but making the header look a bit clumsy.(not having any strong opinion, may be some other way or place :|).
header

Comment (joscollin) [reply src/tools/cephfs/top/cephfs-top:816]: Another way is to have different colour indications for sort & limit. But then it couldn't show the sorted field.
About another place, I think this is the only place in `Info` screens to display the current status. May be, we could make it the same line as `Total Clients` ?

Comment (neesingh-rh) [reply src/tools/cephfs/top/cephfs-top:816]: > Another way is to have different colour indications for sort & limit. But then it couldn't indicate the sorted field. About another place, I think this is the only place in `Info` screens to display the current status. May be, we could make it the same line as `Total Clients` ?

I think there is no better way. Let's keep it as it is, just with small change : s/ Sort=chit, Limit=None/Sort - chit , Limit - None

Comment (joscollin) [reply src/tools/cephfs/top/cephfs-top]: Considering the limitations of what ncurses provides, this way of displaying the settings seems better and much more informative than the alternatives we've considered. Updated the PR as per your comment.

**Review by neesingh-rh — APPROVED** [review]

**Comment by vshankar** [comment]
> This would help the user to identify what are the filters currently applied to the cephfs-top screens at a glance.

I like the idea +1

**Pull request merged**